client/styles/components/_console.scss

```diff
@@ -18,28 +18,53 @@
 	}
 
 	// assign styles to different types of console messages
-	.preview-console__log {
-		@include themify(){
-			color: getThemifyVariable('secondary-text-color');
-		}
+	.preview-console__log, .preview-console__info{
 		flex: 1 0 auto;
+		position: relative;
+
+		.preview-console__log__logged-times, .preview-console__info__logged-times{
```

Comment:
if you change the logged times class to `.preview-console__logged-times` only, you can write this style code pretty concisely:
```
.preview-console__logged-times {
  .preview-console__message--info &, .preview-console__message--log & {
    background-color: $lightsteelblue;
  }
  .preview-console__message--warn & {
    background-color: $orange;
  }
  .preview-console__message--debug & {
    background-color: $dodgerblue;
  }
  .preview-console__message--error & {
    background-color: $red;
  }
}
```

Reply:
Thanks :) This is great